resources/public/api/conf/2.0/schemas/InvestorPayment.get.response.schema.json

```diff
@@ -36,7 +36,7 @@
     "paymentReference" : {
       "description": "The reference of the payment made.",
       "type" : "string",
-      "pattern": "^.{1,12}$",
+      "pattern": "^.{1,16}$",
```

Comment:
Again, does this need comms to go out? I can see how this could be breaking change